ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -13,16 +13,25 @@ struct CreateTaskView: View {
     @ObservedObject private var viewModel: CreateTaskViewModel
     @State var titleInput: String = ""
     @State var date: Date = Date()
+    private var placeHolderTitle: String
     init(viewModel: CreateTaskViewModel) {
         self.viewModel = viewModel
+        switch viewModel.mode {
+        case .createNewTask:
+            self.placeHolderTitle = "Insert Title"
+        case .editExistingTask(_):
+            self.placeHolderTitle = viewModel.title
+        case .none:
+            self.placeHolderTitle = "Insert Title"
+        }
     }
     var body: some View {
         NavigationStack {
             VStack(alignment: .leading, spacing: 21) {
                 VStack(alignment: .leading, spacing: 4) {
                     Text("Task Name")
                         .foregroundColor(Color(.primaryText))
-                    TextField("Insert Title", text: $titleInput)
+                    TextField(placeHolderTitle, text: $titleInput)
```

Comment:
we should not change the placeholder title to the task title since the placeholder is something that is shown when a text field is empty. it would confuse the user when he or she will clear the textfield's content while editing a task and will see the previous task name